src/main/java/com/mitchej123/hodgepodge/HodgepodgeEventHandler.java

```diff
@@ -19,4 +37,105 @@ public void onZombieAidSummon(ZombieEvent.SummonAidEvent event) {
     public void setAidTriggerDisabled(boolean disableAidTrigger) {
         xuDisableAidTrigger = disableAidTrigger;
     }
+
+    @SideOnly(Side.CLIENT)
+    @SubscribeEvent(priority = EventPriority.LOWEST)
+    public void getTooltip(ItemTooltipEvent event) {
+        String producesPollutionFormat = "Produces %d Pollution/Second";
+
+        if (Common.config.furnacesPollute) {
+            String furnacePollution = String.format(producesPollutionFormat, Common.config.furnacePollutionAmount);
+            if (GT_Utility.areStacksEqual(event.itemStack, new ItemStack(Blocks.furnace))
+                    || GT_Utility.areStacksEqual(event.itemStack, IC2Items.getItem("ironFurnace"))) {
+                event.toolTip.add(furnacePollution);
+            }
+
+            if (Compat.isThaumcraftPresent()) {
+                if (GT_Utility.areStacksEqual(event.itemStack, ItemApi.getBlock("blockStoneDevice", 0))) {
+                    event.toolTip.add(furnacePollution);
+                }
+            }
+
+            if (Compat.isThaumicBasesPresent()) {
+                if (GT_Utility.areStacksEqual(
+                        event.itemStack,
+                        GT_ModHandler.getModItem("thaumicbases", "advAlchFurnace", 1, 0))) {
+                    event.toolTip.add(furnacePollution);
+                }
+            }
+        }
+
+        if (Compat.isRailcraftPresent() && Common.config.railcraftPollutes) {
+            String multiProducesPollutionFormat = "A complete Multiblock produces %d Pollution/Second";
+
+            if (GT_Utility.areStacksEqual(
+                    event.itemStack,
+                    new ItemStack(
+                            RailcraftBlocks.getBlockMachineBeta(),
```

Comment:
Do not add additional dependencies. Use `GT_ModHandler.getModItem`.

Reply:
Use now only `GT_ModHandler.getModItem` for modded items.